test/integration/fixtures/install-variants/bower-registry/bower.herokuapp.com/lookup/webcomponentsjs_7bba9

```diff
@@ -0,0 +1 @@
+{"expires":1489941177396,"value":{"type":"alias","url":"https://github.com/Polymer/webcomponentsjs.git"}}
```

Comment:
Good find, I double checked and expired lookups don't affect offline installs, only online refreshes of the cache. Still did it anyway, and added a note about it to the README instructions in case it changes.